application/models/Interfaces/PermissionInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface PermissionInterface
+{
+    public function getOwnerId();
+    public static function getPermissionData($key = null);
```

Comment:
PHPUnit cannot mock static functions. Avoid if possible.

Reply:
It's really a static system fact : it's definition array …

Reply:
So it needs no mocking?

Reply:
No readon : https://github.com/LimeSurvey/LimeSurvey/blob/4b569fc2e6d55733176f782695dd84fbf4fd8206/application/models/Survey.php#L2142